app-forensics/pebear/pe_bear-0.6.5.2.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="PE-bear reverse engineering tool"
+DESCRIPTION="Portable Executable reversing tool with a friendly GUI"
+HOMEPAGE="https://hshrzd.wordpress.com/pe-bear"
+COPYRIGHT="2022-2023 hasherezade"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/hasherezade/pe-bear/archive/refs/tags/v$portVersion.zip"
+CHECKSUM_SHA256="bbd46e4af231ae5257538e8157a077b2fe5843a5dc18beaaaabbae928bfa1025"
+SOURCE_DIR="pe-bear-$portVersion"
+srcGitSubHash1="e2c08098410c85e5546b77a2785444bbc9e0e21b"
+SOURCE_URI_2="https://github.com/hasherezade/bearparser/archive/$srcGitSubHash1.zip"
+CHECKSUM_SHA256_2="4d85d8207285ae5efa7c016aa62606ca155894fee789f028822f57a33117e590"
+SOURCE_FILENAME_2="bearparser-$srcGitSubHash1.zip"
+srcGitSubHash2="61cdc56159986e6d77be571b04c570cf2c378d13"
+SOURCE_URI_3="https://github.com/capstone-engine/capstone/archive/$srcGitSubHash2.zip"
+CHECKSUM_SHA256_3="821ba752993719617ca4c8b546a79dec876c06eedf1e393e86ee4c6899e00099"
+SOURCE_FILENAME_3="capstone-$srcGitSubHash2.zip"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	pe_bear$secondaryArchSuffix = $portVersion
+	app:PE_bear = $portVersion
+	cmd:pe_bear
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Core$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	# link submodules
+	rm -rf $sourceDir/bearparser $sourceDir/capstone
+	ln -srf $sourceDir2/bearparser-$srcGitSubHash1 $sourceDir/bearparser
+	ln -srf $sourceDir3/capstone-$srcGitSubHash2 $sourceDir/capstone
+	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release
+	make -C build
+}
+
+INSTALL()
+{
+	mkdir -p $binDir $appsDir
```

Comment:
why $binDir ?

Reply:
Because it can also be launched from terminal (this is how I use it on Linux). Maybe I should just place it in $appsDir?

Reply:
@korli sorry I was in a rush this morning and I misunderstood your comment. You are actually right, $binDir is out of place here. My initial idea was to put the binary there too so it can be launched from terminal, but there is really no need. I'll remove it later today. Thanks a lot and sorry for this.